src/main/java/de/tum/in/www1/artemis/web/rest/dto/TutorLeaderboardDTO.java

```diff
@@ -29,13 +29,15 @@
 
     private double averageRating;
 
+    private long numberOfTutorRatings;
+
```

Comment:
why did you decide to use the variable `numberOfTutorRatings` and not `numberOfRatings` anymore? Wouldn't it be more consistent to use the same?